packages/vats/src/core/chain-behaviors.js

```diff
@@ -363,7 +363,7 @@ export const makeChainStorage = async ({
   if (!bridgeManager) {
     console.warn('Cannot support chainStorage without an actual chain.');
     chainStorageP.resolve(null);
-    storageBridgeManagerP.resolve(null);
+    storageBridgeManagerP.resolve(undefined);
```

Comment:
This is a runtime change to a bundle that is already on chain... and in a vat where upgrade is an open problem.
I prefer that we leave this as `null` and use `@ts-expect-error`.

p.s. re the open problem of upgrade of the bootstrap vat: I don't see an issue to reference. Perhaps that's appropriate since we have no plans to address it, but it would be convenient in situations such as this. We have an [`'audit bootstrap exports'` test](https://github.com/Agoric/agoric-sdk/blob/386dc7cf179bf610841dec4a0b1f55d5d5c50878/packages/boot/test/bootstrapTests/test-vaults-upgrade.js#L160-L161) that might be a useful reference, but it doesn't actually document its motivation. That test seems to originate from #4548, which we closed even though a few are _not_ stored durably.

Reply:
Yeah I agree I'd prefer not having runtime changes, however minimal, in a types refactor PR.